docs/release-notes/flink-1.10.md

```diff
@@ -0,0 +1,377 @@
+directories appearing first.
+
+#### Removal of --yn/--yarncontainer Command Line Options ([FLINK-12362](https://issues.apache.org/jira/browse/FLINK-12362))
+The Flink CLI no longer supports the deprecated command line options
+`-yn/--yarncontainer`, which were used to specify the number of containers to
+start on YARN. This option has been deprecated since the introduction of
+[FLIP-6](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=65147077).
+All Flink users are advised to remove this command line option.
+
+#### Mesos Integration will reject expired Offers faster ([FLINK-14029](https://issues.apache.org/jira/browse/FLINK-14029))
+Flink's Mesos integration now rejects all expired offers instead of only 4.
+This improves the situation where Fenzo holds on to a lot of expired offers
+without giving them back to the Mesos resource manager.
+
+#### Scheduler Rearchitecture ([FLINK-14651](https://issues.apache.org/jira/browse/FLINK-14651))
+Flink's scheduler was refactored with the goal of making scheduling strategies
+customizable in the future. Users that experience issues related to scheduling
+can fallback to the legacy scheduler by setting `jobmanager.scheduler: legacy`
+in their `flink-conf.yaml`.
+
+
+### Memory Management
+#### New Task Executor Memory Model ([FLINK-13980](https://issues.apache.org/jira/browse/FLINK-13980))
+With
+[FLIP-49](https://cwiki.apache.org/confluence/display/FLINK/FLIP-49%3A+Unified+Memory+Configuration+for+TaskExecutors),
+a new memory model has been introduced for the task executor. New configuration
+options have been introduced to control the memory consumption of the task
+executor process. This affects all types of deployments: standalone, YARN,
+Mesos, and the new active Kubernetes integration. The memory model of the job
+manager process has not been changed yet but it is planned to be updated as
+well.
+
+If you try to reuse your previous Flink configuration without any adjustments,
+the new memory model can result in differently computed memory parameters for
+the JVM and, thus, performance changes.
+
+Please check the user documentation <!-- TODO: insert link --> for more details.
+
+##### Deprecation and breaking changes
+The following options have been removed and have no effect anymore:
+
+<table class="table">
+  <thead>
+    <tr>
+      <th class="text-left" style="width: 30%">Deprecated/removed config option</th>
+      <th class="text-left">Note</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>taskmanager.memory.fraction</td>
+      <td>
+        Check also the description of the new option
+        <code class="highlighter-rouge">taskmanager.memory.managed.fraction</code>
+        but it has different semantics and the value of the deprecated option
+        usually has to be adjusted
+      </td>
+    </tr>
+    <tr>
+      <td>taskmanager.memory.off-heap</td>
+      <td>On-heap managed memory is no longer supported</td>
```

Comment:
Which is it that's no longer supported: off-heap or on-heap?

Reply:
on heap managed memory. Hence, there is no need for this config option anymore.

Reply:
Without having more context, I misinterpreted this entry as a typo. If you don't know that this has been a boolean that selects between two mutually exclusive options, then one can get confused. I suggest adding a few more words to make this more obvious. Perhaps something like "Support for on-heap managed memory has been removed, leaving off-heap managed memory as the only possibility." or "This configuration item is no longer meaningful, since there no longer an alternative to using off-heap managed memory."

Reply:
fixed in 62b54f9b26b8546dedb250788596aa9a1fe649c8